cfme/configure/access_control.py

```diff
@@ -168,149 +248,295 @@ def description(self):
+    VIEW = EditTagsUserView
     prerequisite = NavigateToSibling('Details')
 
     def step(self):
-        tb_select('Edit this User')
+        self.view.policy.item_select("Edit 'My Company' Tags for this User")
+
+
+class GroupForm(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    ldap_groups_for_user = BootstrapSelect(id='ldap_groups_user')
+    description_txt = Input(name='description')
+    lookup_ldap_groups_chk = Checkbox(name='lookup')
+    role_select = BootstrapSelect(id='group_role')
+    group_tenant = BootstrapSelect(id='group_tenant')
+    user_to_look_up = Input(name='user')
+    username = Input(name='user_id')
+    password = Input(name='password')
+
+    tag = SummaryFormItem('Smart Management', 'My Company Tags')
+
+    cancel_button = Button('Cancel')
+    retrieve_button = Button('Retrieve')
+
+    @View.nested
+    class my_company_tags(Tab):     # noqa
+        TAB_NAME = "My Company Tags"
+        tag_tree = CheckableBootstrapTreeview('tags_treebox')
+
+    @View.nested
+    class hosts_and_clusters(Tab):      # noqa
+        TAB_NAME = "Hosts & Clusters"
+        hosts_clusters_tree = CheckableBootstrapTreeview('hac_treebox')
+
+    @View.nested
+    class vms_and_templates(Tab):       # noqa
+        TAB_NAME = "VMs & Templates"
+        vms_templates_tree = CheckableBootstrapTreeview('vat_treebox')
+
+
+class AddGroupView(GroupForm):
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == "Adding a new Group"
+        )
+
+
+class DetailsGroupView(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == 'EVM Group "{}"'.format(self.context['object'].description)
+        )
+
+
+class EditGroupView(GroupForm):
+    save_button = Button("Save")
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == 'Editing Group "{}"'.format(self.context['object'].description)
+        )
+
+
+class AllGroupView(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == 'Access Control EVM Groups'
+        )
+
+
+class EditGroupSequenceView(GroupForm):
+    title = Text('#explorer_title_text')
+
+    group_order_selector = UpDownSelect(
+        '#seq_fields',
+        './/a[@title="Move selected fields up"]/img',
+        './/a[@title="Move selected fields down"]/img')
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+    cancel_button = Button('Cancel')
```

Comment:
EditGroupSequenceView doesn't have multiple elements from GroupForm, so it inherits ConfigurationView which doesn't have any of these buttons